patrol_node:
  ros__parameters:
    # 机器人初始位姿 (x, y, yaw in radians)
    initial_pose:
      x: 1.0
      y: 0.0
      yaw: 0.0

    # 巡逻点列表
    # 格式: ["x,y,yaw_in_degrees", "x,y,yaw_in_degrees", ...]
    patrol_points: [
      "3.2, -1.0, 0.0",   # 目标点1
      "4.7, -4.7, 0.0",   # 目标点2
      "3.6, 2.8, 0.0",   # 目标点3
      "-2.4, -3.0, 0.0",     # 目标点4 
      "-7.7, -4.8, 0.0",      # 目标点5
      "-6.7, 1.9, 0.0",      # 目标点6 
      "0.0, 0.0, 0.0"      # 目标点7（返回原点） 
    ]